File: problems/round-1/problem-2/bugtrons-2.cs
using System;

class bugtrons2
{

static int maxProfitofBitcoin(int []rate, int begin, int over)
{
  // Write your code here 
}
// Driver code
public static void Main(String[] args)
{
	int []rate = { 100, 180, 260, 310,
					40, 535, 695 };
	int n = rate.Length;

	Console.Write(maxProfitofBitcoin(rate, 0, n - 1));
}
}

File: problems/round-1/problem-2/bugtrons-2.cpp
#include <bits/stdc++.h>
using namespace std;

int maxProfitofBitcoin(int rate[], int start, int end)
{
  // Write your code here 
}

// Driver code
int main()
{
	int rate[] = { 130, 210, 290, 340, 70, 565, 625 };
	int n = sizeof(rate) / sizeof(rate[0]);

	cout << maxProfitofBitcoin(rate, 0, n - 1);

	return 0;
}
